tests/invariants/pass/amm-full.act

```diff
@@ -5,16 +5,12 @@ creates
     mapping(address => uint) balanceOf :=  [CALLER := _supply]
 
 constructor of Amm
-interface constructor(Token t0, Token t1)
+interface constructor(uint s0, uint s1)
 
 creates
 
-    Token token0 := t0
-    Token token1 := t1
-
-invariants
-
-    pre(token0.balanceOf[THIS]) * pre(token1.balanceOf[THIS]) <= post(token0.balanceOf[THIS]) * post(token1.balanceOf[THIS])
+    Token token0 := Token(s0)
```

Comment:
This is very close to the solidity syntax for casting an addressinto a contract type, and would read that way to may solidity developers IMO. What do you think about introducing a `create` keyword to make the construction of a new contract explicit here?

Reply:
I think that's great, I will add a new keyword

Reply:
Done!